### PR TITLE
Test infra refactor

### PR DESCRIPTION
**Pull request by zefhemel**

### CODE REVIEW

File: testinfra/integration_test.go [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+func TestSQS(t *testing.T) {
+	infra := setupInfra(t, "SQS")
+	sqsConn := testinfra.ConnectSQSLocal()
+	queueURL := infra.GetQueueURL("SQSQueue")
+	// Creating temp file for lambda function to write to
+	f, err := ioutil.TempFile("", "test-sqs")
+	f.Close()
+	defer os.Remove(f.Name())
+	if err != nil {
```
Comment (tlakomy) [inline testinfra/integration_test.go]: Won't `f.Close()` break before we get to this line in case of an `err` ?

**Comment by tlakomy** [comment]
Review finished, LGTM - some minor comments left :)

File: serverless_test.yml [hunk 2]
```diff
@@ -30,10 +30,25 @@ provider:
...
+    # Ignored by test runner
+    handler: bin/sqs-handler
+    # This is ignored serverless, but by our test runner to uses it to build and run
+    go-src: test-functions/sqs-handler
+    memorySize: 128
+    events:
+    - sqs:
+        arn:
+          Fn::GetAtt:
+          - TestQueue
```
[the remark this answers is not in the document]
Comment (zefhemel) [reply serverless_test.yml]: No, this is a resource name, not a queue name, so it won't change.

File: testinfra/integration_test.go (hunk 1 above)
Comment (zefhemel) [reply testinfra/integration_test.go]: yes...

Comment (zefhemel) [reply testinfra/integration_test.go]: Fixed

**Pull request merged**